scenario/mocking.py

```diff
@@ -20,6 +20,7 @@
     cast,
 )
 
+import ops
```

Comment:
I really don't like mixing `import X` and `from X import Y` in the same module. I'm not sure if this is just a personal bugbear, though - I certainly do see it in a lot of code.

I like `import ops` more than `from ops import`, but I would say that this is probably better if `CloudSpec` is added on the next line instead, but, again, maybe this is just a me thing.

Reply:
In our tutorial I think we are encouraging `import ops` instead of importing specific things, that's why I used it. Shall I keep it or not?

Reply:
I will revert it so that it's consistent in this PR and make another small PR to bring all imports to the latest suggested way.